### PR TITLE
Toasts

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
Add Dismissable Property
#78

### CODE REVIEW

**Review by BAL4-R — APPROVED** [review]

**Review by rthanga1 — CHANGES_REQUESTED** [review]
isDismissable should work only when the action label is null or empty.
1. So when the action label is given, then the action button will be displayed. Clicking that will close the toast and also calls the action.
2. When the action label is empty, then X button will be shown only when the isDismissable is true.

So basically, I think we should rephrase this. Let's discuss once you are in.

**Review by rthanga1 — APPROVED** [review]

**Review by rthanga1 — DISMISSED** [review]

**Review by Sofiya-kumar — CHANGES_REQUESTED** [review]
Please have the toast as single file

**Review by Sofiya-kumar — CHANGES_REQUESTED** [review]
Please have the toast as single file

File: Trimble.Modus.Components/Controls/Toast/TMToast.cs [hunk 1]
```diff
@@ -5,25 +5,26 @@ namespace Trimble.Modus.Components.Controls.Toast
 {
     public class TMToast
     {
-        PopupNavigation popupNavigation;
-
-        public TMToast()
+        private PopupNavigation popupNavigation;
```
Comment (Sofiya-kumar) [inline Trimble.Modus.Components/Controls/Toast/TMToast.cs:8]: Add regions

**Review by Sofiya-kumar — CHANGES_REQUESTED** [review]

File: Trimble.Modus.Components/Controls/Toast/TMToast.cs (hunk 1 above)
Comment (ElishaSamPeterPrabhu) [reply Trimble.Modus.Components/Controls/Toast/TMToast.cs:9]: Added

**Comment by ElishaSamPeterPrabhu** [comment]
> Please have the toast as single file

Using a single file will allow the user to have access to the toasts contents page and can display it as a whole page , unlike now were they can access it only via show()

**Review by Sofiya-kumar — APPROVED** [review]

**Review by rthanga1 — APPROVED** [review]

**Review by rthanga1 — APPROVED** [review]

**Pull request merged**